collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinSdkAdAdapter.kt

```diff
@@ -27,44 +28,77 @@ class BitmovinSdkAdAdapter(val bitmovinPlayer: BitmovinPlayer, val adAnalytics:
     private val adMapper: AdMapper = AdMapper()
     private val adBreakMapper: AdBreakMapper = AdBreakMapper()
     private val adQuartileFactory: AdQuartileFactory = AdQuartileFactory()
+    private val TAG = "BitmovinSdkAdAdapter"
 
     private val onAdStartedListener = OnAdStartedListener {
-        val ad = it.ad ?: return@OnAdStartedListener
-        adAnalytics.onAdStarted(adMapper.fromPlayerAd(ad))
+        try {
+            val ad = it.ad ?: return@OnAdStartedListener
+            adAnalytics.onAdStarted(adMapper.fromPlayerAd(ad))
+        } catch (e: Exception) {
+            Log.d(TAG, "On Ad Started", e)
```

Comment:
Can't we just directly log the exception? Or some more descriptive text - because the Stacktrace should contain the location anyways so no need to repeat ourselves